gradle.properties

```diff
@@ -1,14 +1,14 @@
 org.gradle.caching=true
 group=io.ballerina
-version=2.0.0-SNAPSHOT
+version=2.0.1-SNAPSHOT
 
 # Client Native Version
-clientNativeVersion=1.0.0-SNAPSHOT
+clientNativeVersion=1.0.1-SNAPSHOT
 # Mark this as false to skip publishing the client native artifacts
-clientNativePublish=false
+clientNativePublish=true
```

Comment:
why do we re-enable this?

Reply:
Ah will fix that 👍🏼

Reply:
Fixed via [172b9bc](https://github.com/ballerina-platform/openapi-tools/pull/1708/commits/172b9bccc3e70c10c5790145c2330e7444b2e39c)